spce/spce.go

```diff
@@ -0,0 +1,141 @@
+package spce
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type SPCE struct {
+	rw io.ReadWriter
+	channel string
+}
+
+// NewSPCE creates a new SPCE instance with the given ReadWriter and channel address in the format of two hexadezimal characters
+// The default channel address is "05"
+func NewSPCE(rw io.ReadWriter, channel string) *SPCE {
+	// If the channel address is not provided, use the default address "05"
+	if channel == "" {
+		channel = "05"
+	}
+	return &SPCE{rw: rw, channel: channel}
+}
+
+func (s *SPCE) Outputs() (map[string]float64, error) {
```

Comment:
What motivated you to use `Outputs()` instead of `Current()`, `Pressure()`, and `Voltage()`?